packages/sdk-components-react/src/xml-element.tsx

```diff
@@ -0,0 +1,32 @@
+import { componentAttribute } from "@webstudio-is/react-sdk";
+import { Children, forwardRef, type ElementRef, type ReactNode } from "react";
+
+export const defaultTag = "div";
+
+// We don't want to enable all tags because Box is usually a container and we have specific components for many tags.
+type Props = {
+  tag: string;
+  children: ReactNode;
+} & Record<typeof componentAttribute, string>;
+
+export const XmlElement = forwardRef<ElementRef<"div">, Props>(
+  ({ tag = "element", children, ...props }, ref) => {
```

Comment:
what's "element" as a tag name?

Reply:
Don't get, here can be any default, why not element

Reply:
We will deciede later for now will merge as is

Reply:
what is element doing in case the default is applied?

Reply:
when is it possible to not get a tag and land on default?
what's the intention of having a default here, who will see it?

Reply:
seems like it should be possible to not have a tag provided and right now I can only imagine an empty string there